client/src/components/AddNewProfile.js

```diff
@@ -32,6 +31,37 @@ const AddNewProfile = () => {
     });
   };
 
+  const profileTypeChange = (event, data) => {
+    updateField("profile_type", data.value);
+  };
+
+  const statusChange = (event, data) => {
+    updateField("status", data.value);
+  };
+
+  const dateChange = (event, data) => {
+    updateField("join_date", data.value);
+  };
```

Comment:
```suggestion
  const handleDropdownAndDateChange = (event, data) => {
    updateField(data.name, data.value);
  };
```
Nice job creating `updateField` to deal with two types of updating. We can simplify this even more so let's do that!

If we add `name="profile_type"`, `name="status"`, and `name="join_date"` to the `<Dropdown>` and `<SemanticDatepicker>` components, then they become available as `data.name`. We can then convert these 3 functions into one which can deal with these 3 inputs.